src/main/java/org/traccar/database/DeviceManager.java

```diff
@@ -325,7 +325,7 @@ public Position getLastPosition(long deviceId) {
     @Override
     public boolean lookupAttributeBoolean(
             long deviceId, String attributeName, boolean defaultValue, boolean lookupConfig) {
-        Object result = lookupAttribute(deviceId, attributeName, lookupConfig);
+        Object result = lookupAttribute(deviceId, attributeName, true, lookupConfig);
```

Comment:
Passing `true` every time changes behavior of these functions.

Reply:
You're right, I coded it that way for the first commit, and never looked back for the second. I think `!lookupConfig` will make the trick.

Reply:
Now, do you think we should override the methods that calls `lookupAttribute`, or modify every call to it?

Reply:
I think modify every call.

Reply:
I agree. Ok, let's do it. Thanks.